experiments/long_runs/land.jl

```diff
@@ -433,3 +433,8 @@ mktempdir(root_path) do tmpdir
         run(Cmd([unite, figures..., joinpath(root_path, "figures.pdf")]))
     end
 end
+
```

Comment:
I think we should add this instead to snowy_land.jl, and run for two years.

Reply:
Is it fine to change it to two years even if the CI will take longer?